javascript/ql/test/query-tests/Security/CWE-754/UnvalidatedDynamicMethodCallGood4.js

```diff
@@ -0,0 +1,20 @@
+var express = require('express');
+var app = express();
+
+var actions = new Map();
+actions.put("play", function play(data) {
+    // ...
+});
+actions.put("pause", function pause(data) {
+    // ...
+});
+
+app.get('/perform/:action/:payload', function(req, res) {
+    if (typeof actions.get(req.params.action) === 'function') {
+        let action = actions.get(req.params.action); // OK but flagged [INCONSISTENCY]
```

Comment:
This is flagged by our query. Is this expected?

Reply:
It's a limitation of the query, I think. In general it's hard to tell that these two calls to `get` must return the same value and hence the first one sanitises the second.

Reply:
Yep, that sounds reasonable.